daq/host.py

```diff
@@ -595,20 +593,18 @@ def _docker_callback(self, return_code=None, exception=None):
         self._monitor_cleanup()
         failed = return_code or exception
         state = MODE.MERR if failed else MODE.DONE
-        self.record_result(self.test_name, state=state, code=return_code, exception=exception)
-        if exception:
-            self._report.accumulate(self.test_name, {ResultType.EXCEPTION: str(exception)})
-        self._report.accumulate(self.test_name, {ResultType.RETURN_CODE: return_code})
-        self._report.accumulate(self.test_name, {ResultType.MODULE_CONFIG: self._loaded_config})
         report_path = os.path.join(self._host_tmp_path(), 'report.txt')
         activation_log_path = os.path.join(self._host_dir_path(), 'activate.log')
         module_config_path = os.path.join(self._host_tmp_path(), self._MODULE_CONFIG)
+        remote_paths = {}
         for result_type, path in ((ResultType.REPORT_PATH, report_path),
                                   (ResultType.ACTIVATION_LOG_PATH, activation_log_path),
                                   (ResultType.MODULE_CONFIG_PATH, module_config_path)):
             if os.path.isfile(path):
                 self._report.accumulate(self.test_name, {result_type: path})
-                self._gcp.upload_file(path, self._get_unique_upload_path(path))
+                remote_paths[result_type.value] = self._upload_file(path)
+        self.record_result(self.test_name, state=state, code=return_code, exception=exception, \
```

Comment:
no line continuation

Reply:
I can't figure out how to make everything fit under 100 characters without making the code convoluted. It's 12 characters over the limit.